src/mame/misc/playcenter.cpp

```diff
@@ -86,21 +88,29 @@ INPUT_PORTS_END
 
 void playcenter_state::playcenter(machine_config &config)
 {
-	PENTIUM(config, m_maincpu, 166'000'000); // Actually an AMD K6 or an Intel Celeron, frequency unknown
+	PENTIUM(config, m_maincpu, 166'000'000); // Actually an AMD K6, AMD K6-2 or Intel Celeron
 	m_maincpu->set_addrmap(AS_PROGRAM, &playcenter_state::mem_map);
 
 	PCI_ROOT(config, "pci", 0);
 	// ...
 }
 
+// 'Epox' version. Windows 98 SE.
+ROM_START(plycntre3)
+	ROM_REGION32_LE(0x40000, "bios", 0) // BIOS date: 03/13/2001
+	ROM_LOAD("vp4f1313.bin", 0x00000, 0x40000, CRC(bd4b155f) SHA1(3eafe71e89bf84b72a42e933187676fe08db0492))
+
+	DISK_REGION( "ide:0:hdd" )
+	DISK_IMAGE("playcenter_e_14_0_t", 0, SHA1(b7c7fce1522e64fc5132f223d8a73555c24002a9)) // From an operator, may contain user data. Dump contains a raw image for a (c) 1998 Trident video card (C:\videorom.bin)
```

Comment:
You can generally detect what video card is by digging the `PCIR` signature inside the binary and check next 4 bytes byteswapped. i.e. `PCIR\de\10\00\01` is a nVidia `10de` GeForce 256 SDR `0100`.
Being likely a Trident Blade MVP4 should be `PCIR\23\10\80\98`

Reply:
Done